src/Tabs/Tabs.js

```diff
@@ -377,4 +377,3 @@ class Tabs extends Component {
 }
 
 export default withWidth()(Tabs);
-export { Tabs as PureTabs }; // for testing purposes
```

Comment:
On second look, it appears (although you did it far cleaner than in my attempts) that I was actually using dive() correctly.  It's your changes to withWidth that allowed this to work.  Either way, glad to have learned something today 😀